<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:w="urn:schemas-microsoft-com:office:word"
xmlns:st1="urn:schemas-microsoft-com:office:smarttags"
xmlns="http://www.w3.org/TR/REC-html40">

<head>
<meta http-equiv=Content-Type content="text/html; charset=iso-8859-1">
<meta name=ProgId content=Word.Document>
<meta name=Generator content="Microsoft Word 11">
<meta name=Originator content="Microsoft Word 11">
<link rel=File-List href="tutorial_files/filelist.xml">
<link rel=Edit-Time-Data href="tutorial_files/editdata.mso">
<!--[if !mso]>
<style>
v\:* {behavior:url(#default#VML);}
o\:* {behavior:url(#default#VML);}
w\:* {behavior:url(#default#VML);}
.shape {behavior:url(#default#VML);}
</style>
<![endif]-->
<title>Network Component Analysis Tutorial</title>
<o:SmartTagType namespaceuri="urn:schemas-microsoft-com:office:smarttags"
 name="City"/>
<o:SmartTagType namespaceuri="urn:schemas-microsoft-com:office:smarttags"
 name="place"/>
<o:SmartTagType namespaceuri="urn:schemas-microsoft-com:office:smarttags"
 name="country-region"/>
<!--[if gte mso 9]><xml>
 <o:DocumentProperties>
  <o:Author>Simon Galbraith</o:Author>
  <o:Template>Normal</o:Template>
  <o:LastAuthor>Simon Galbraith</o:LastAuthor>
  <o:Revision>42</o:Revision>
  <o:TotalTime>19</o:TotalTime>
  <o:Created>2005-11-10T21:38:00Z</o:Created>
  <o:LastSaved>2005-11-15T18:12:00Z</o:LastSaved>
  <o:Pages>1</o:Pages>
  <o:Words>1854</o:Words>
  <o:Characters>10569</o:Characters>
  <o:Company>ucla</o:Company>
  <o:Lines>88</o:Lines>
  <o:Paragraphs>24</o:Paragraphs>
  <o:CharactersWithSpaces>12399</o:CharactersWithSpaces>
  <o:Version>11.6568</o:Version>
 </o:DocumentProperties>
</xml><![endif]--><!--[if gte mso 9]><xml>
 <w:WordDocument>
  <w:Zoom>150</w:Zoom>
  <w:SpellingState>Clean</w:SpellingState>
  <w:GrammarState>Clean</w:GrammarState>
  <w:ValidateAgainstSchemas/>
  <w:SaveIfXMLInvalid>false</w:SaveIfXMLInvalid>
  <w:IgnoreMixedContent>false</w:IgnoreMixedContent>
  <w:AlwaysShowPlaceholderText>false</w:AlwaysShowPlaceholderText>
  <w:BrowserLevel>MicrosoftInternetExplorer4</w:BrowserLevel>
 </w:WordDocument>
</xml><![endif]--><!--[if gte mso 9]><xml>
 <w:LatentStyles DefLockedState="false" LatentStyleCount="156">
 </w:LatentStyles>
</xml><![endif]--><!--[if !mso]><object
 classid="clsid:38481807-CA0E-42D2-BF39-B33AF135CC4D" id=ieooui></object>
<style>
st1\:*{behavior:url(#ieooui) }
</style>
<![endif]-->
<style>
<!--
 /* Style Definitions */
 p.MsoNormal, li.MsoNormal, div.MsoNormal
	{mso-style-parent:"";
	margin:0in;
	margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:"Times New Roman";
	mso-fareast-font-family:"Times New Roman";}
h1
	{mso-margin-top-alt:auto;
	margin-right:0in;
	mso-margin-bottom-alt:auto;
	margin-left:0in;
	mso-pagination:widow-orphan;
	mso-outline-level:1;
	font-size:24.0pt;
	font-family:"Times New Roman";
	font-weight:bold;}
a:link, span.MsoHyperlink
	{color:blue;
	text-decoration:underline;
	text-underline:single;}
a:visited, span.MsoHyperlinkFollowed
	{color:blue;
	text-decoration:underline;
	text-underline:single;}
p
	{mso-margin-top-alt:auto;
	margin-right:0in;
	mso-margin-bottom-alt:auto;
	margin-left:0in;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:"Times New Roman";
	mso-fareast-font-family:"Times New Roman";}
pre
	{margin:0in;
	margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	tab-stops:45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt;
	font-size:10.0pt;
	font-family:"Courier New";
	mso-fareast-font-family:"Times New Roman";}
span.SpellE
	{mso-style-name:"";
	mso-spl-e:yes;}
span.GramE
	{mso-style-name:"";
	mso-gram-e:yes;}
@page Section1
	{size:8.5in 11.0in;
	margin:1.0in 1.25in 1.0in 1.25in;
	mso-header-margin:.5in;
	mso-footer-margin:.5in;
	mso-paper-source:0;}
div.Section1
	{page:Section1;}
 /* List Definitions */
 @list l0
	{mso-list-id:580409855;
	mso-list-template-ids:-903727790;}
@list l0:level1
	{mso-level-number-format:bullet;
	mso-level-text:\F0B7;
	mso-level-tab-stop:.5in;
	mso-level-number-position:left;
	text-indent:-.25in;
	mso-ansi-font-size:10.0pt;
	font-family:Symbol;}
@list l0:level2
	{mso-level-tab-stop:1.0in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l0:level3
	{mso-level-tab-stop:1.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l0:level4
	{mso-level-tab-stop:2.0in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l0:level5
	{mso-level-tab-stop:2.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l0:level6
	{mso-level-tab-stop:3.0in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l0:level7
	{mso-level-tab-stop:3.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l0:level8
	{mso-level-tab-stop:4.0in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l0:level9
	{mso-level-tab-stop:4.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l1
	{mso-list-id:882450159;
	mso-list-template-ids:24147886;}
@list l1:level1
	{mso-level-number-format:bullet;
	mso-level-text:\F0B7;
	mso-level-tab-stop:.5in;
	mso-level-number-position:left;
	text-indent:-.25in;
	mso-ansi-font-size:10.0pt;
	font-family:Symbol;}
@list l1:level2
	{mso-level-tab-stop:1.0in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l1:level3
	{mso-level-tab-stop:1.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l1:level4
	{mso-level-tab-stop:2.0in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l1:level5
	{mso-level-tab-stop:2.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l1:level6
	{mso-level-tab-stop:3.0in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l1:level7
	{mso-level-tab-stop:3.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l1:level8
	{mso-level-tab-stop:4.0in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l1:level9
	{mso-level-tab-stop:4.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l2
	{mso-list-id:1023242008;
	mso-list-template-ids:-41511170;}
@list l2:level1
	{mso-level-number-format:bullet;
	mso-level-text:\F0B7;
	mso-level-tab-stop:.5in;
	mso-level-number-position:left;
	text-indent:-.25in;
	mso-ansi-font-size:10.0pt;
	font-family:Symbol;}
@list l2:level2
	{mso-level-tab-stop:1.0in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l2:level3
	{mso-level-tab-stop:1.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l2:level4
	{mso-level-tab-stop:2.0in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l2:level5
	{mso-level-tab-stop:2.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l2:level6
	{mso-level-tab-stop:3.0in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l2:level7
	{mso-level-tab-stop:3.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l2:level8
	{mso-level-tab-stop:4.0in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l2:level9
	{mso-level-tab-stop:4.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l3
	{mso-list-id:1991128792;
	mso-list-template-ids:1376976732;}
@list l3:level1
	{mso-level-number-format:bullet;
	mso-level-text:\F0B7;
	mso-level-tab-stop:.5in;
	mso-level-number-position:left;
	text-indent:-.25in;
	mso-ansi-font-size:10.0pt;
	font-family:Symbol;}
@list l3:level2
	{mso-level-tab-stop:1.0in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l3:level3
	{mso-level-tab-stop:1.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l3:level4
	{mso-level-tab-stop:2.0in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l3:level5
	{mso-level-tab-stop:2.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l3:level6
	{mso-level-tab-stop:3.0in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l3:level7
	{mso-level-tab-stop:3.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l3:level8
	{mso-level-tab-stop:4.0in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l3:level9
	{mso-level-tab-stop:4.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l4
	{mso-list-id:2015062519;
	mso-list-template-ids:88121956;}
@list l4:level1
	{mso-level-number-format:bullet;
	mso-level-text:\F0B7;
	mso-level-tab-stop:.5in;
	mso-level-number-position:left;
	text-indent:-.25in;
	mso-ansi-font-size:10.0pt;
	font-family:Symbol;}
@list l4:level2
	{mso-level-number-format:bullet;
	mso-level-text:o;
	mso-level-tab-stop:1.0in;
	mso-level-number-position:left;
	text-indent:-.25in;
	mso-ansi-font-size:10.0pt;
	font-family:"Courier New";
	mso-bidi-font-family:"Times New Roman";}
@list l4:level3
	{mso-level-tab-stop:1.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l4:level4
	{mso-level-tab-stop:2.0in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l4:level5
	{mso-level-tab-stop:2.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l4:level6
	{mso-level-tab-stop:3.0in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l4:level7
	{mso-level-tab-stop:3.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l4:level8
	{mso-level-tab-stop:4.0in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l4:level9
	{mso-level-tab-stop:4.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
ol
	{margin-bottom:0in;}
ul
	{margin-bottom:0in;}
-->
</style>
<!--[if gte mso 10]>
<style>
 /* Style Definitions */
 table.MsoNormalTable
	{mso-style-name:"Table Normal";
	mso-tstyle-rowband-size:0;
	mso-tstyle-colband-size:0;
	mso-style-noshow:yes;
	mso-style-parent:"";
	mso-padding-alt:0in 5.4pt 0in 5.4pt;
	mso-para-margin:0in;
	mso-para-margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:10.0pt;
	font-family:"Times New Roman";
	mso-ansi-language:#0400;
	mso-fareast-language:#0400;
	mso-bidi-language:#0400;}
</style>
<![endif]-->
</head>

<body lang=EN-US link=blue vlink=blue style='tab-interval:.5in'>

<div class=Section1>

<div>

<h1><span style='font-size:14.0pt;font-family:Arial'>Network Component Analysis
Tutorial<u1:p></u1:p></span><span style='font-family:Arial'><o:p></o:p></span></h1>

<pre><span style='font-family:Arial'>Simon J. Galbraith<br style='mso-special-character:
line-break'>
<![if !supportLineBreakNewLine]><br style='mso-special-character:line-break'>
<![endif]><o:p></o:p></span></pre><pre><span style='font-family:Arial'><o:p>&nbsp;</o:p></span></pre><pre><span
style='font-family:Arial'><a href="mailto:sgalbrai@cs.ucla.edu">sgalbrai@cs.ucla.edu</a><o:p></o:p></span></pre><pre><span
style='font-family:Arial'><o:p>&nbsp;</o:p></span></pre><pre><span
class=SpellE><span style='font-family:Arial'>Linh</span></span><span
style='font-family:Arial'> Tran<br style='mso-special-character:line-break'>
<![if !supportLineBreakNewLine]><br style='mso-special-character:line-break'>
<![endif]><o:p></o:p></span></pre><pre><span style='font-family:Arial'><o:p>&nbsp;</o:p></span></pre><pre><span
style='font-family:Arial'><a href="mailto:ltran@seas.ucla.edu">ltran@seas.ucla.edu</a><o:p></o:p></span></pre><pre><span
style='font-family:Arial'><o:p>&nbsp;</o:p></span></pre><pre><span
style='font-family:Arial'>Professor James C. Liao<span style='mso-tab-count:
2'>                        </span><br style='mso-special-character:line-break'>
<![if !supportLineBreakNewLine]><br style='mso-special-character:line-break'>
<![endif]><o:p></o:p></span></pre><pre><span style='font-family:Arial'><o:p>&nbsp;</o:p></span></pre><pre><span
style='font-family:Arial'><a href="mailto:liaoj@seas.ucla.edu">liaoj@seas.ucla.edu</a><span style='mso-spacerun:yes'>   </span><span
style='mso-tab-count:1'>             </span><a
href="http://www.seas.ucla.edu/%7Eliaoj">http://www.seas.ucla.edu/~liaoj&nbsp;</a><o:p></o:p></span></pre>

<p style='tab-stops:45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt'><b><span
style='font-size:13.5pt;font-family:Arial'><br>
Introduction<br>
</span></b><span style='font-family:Arial'><br>
The network component analysis (NCA) toolbox is organized as a workflow.&nbsp;
This means that specific steps have to occur in order for the program to run
correctly.&nbsp; The graphical user-interface (GUI) portion of toolbox is
designed to run one NCA analysis at a time, and allow you to visualize the
results. This tutorial is designed to get you to introduce you to NCA and to
use the NCA Toolbox.&nbsp; Most importantly, the GUI toolbox only works with
MATLAB 7.0, if you require a version of the GUI to work with previous editions
of MATLAB please contact us and we will make one available.&nbsp; However, you
should be able to run the non-GUI command line NCA programs from MATLAB prompt
with previous versions of MATLAB.</span><b><span style='font-size:13.5pt;
font-family:Arial'><br style='mso-special-character:line-break'>
<![if !supportLineBreakNewLine]><br style='mso-special-character:line-break'>
<![endif]><o:p></o:p></span></b></p>

<p style='tab-stops:45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt'><b><span
style='font-size:13.5pt;font-family:Arial'>Tutorial Contents:</span></b><span
style='font-family:Arial'><o:p></o:p></span></p>

<ul type=disc>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l2 level1 lfo1;tab-stops:list .5in'><span style='font-family:
     Arial'>Installing and Running the NCA Toolbox Version 1.0 <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l2 level1 lfo1;tab-stops:list .5in'><span style='font-family:
     Arial'>Demo NCA Analysis Session <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l2 level1 lfo1;tab-stops:list .5in'><span style='font-family:
     Arial'>Analyzing Spectra Data <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l2 level1 lfo1;tab-stops:list .5in'><span style='font-family:
     Arial'>Input Data File Formats and Example Data Files <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l2 level1 lfo1;tab-stops:list .5in'><span style='font-family:
     Arial'>How to analyze organisms not listed in the NCA Toolbox <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l2 level1 lfo1;tab-stops:list .5in'><span style='font-family:
     Arial'>A note on NCA <span class=SpellE>Genebank</span> Formats <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l2 level1 lfo1;tab-stops:list .5in'><span style='font-family:
     Arial'>Common Problems/Questions and How to Get Help <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l2 level1 lfo1;tab-stops:list .5in'><span style='font-family:
     Arial'>MATLAB variable description in the Global Workspace<o:p></o:p></span></li>
</ul>

<p class=MsoNormal style='tab-stops:45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt'><span
style='font-family:Arial'><o:p>&nbsp;</o:p></span></p>

<div class=MsoNormal align=center style='text-align:center;tab-stops:45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt'><span
style='font-family:Arial'>

<hr size=2 width="100%" align=center>

</span></div>

<pre><b><span style='font-size:14.0pt;font-family:Arial'>I. Installing and Running the NCA Toolbox Version 1.0:<u1:p></u1:p></span></b><span
style='font-family:Arial'><o:p></o:p></span></pre><pre><b><span
style='font-size:12.0pt;font-family:Arial'>Instructions for Windows XP:<u1:p></u1:p></span></b><span
style='font-family:Arial'><o:p></o:p></span></pre>

<ul type=disc>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l0 level1 lfo2;tab-stops:list .5in'><span style='font-family:
     Arial'>Unzip the NCA Toolbox Release ZIP File and extract all files to a
     directory. <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l0 level1 lfo2;tab-stops:list .5in'><span style='font-family:
     Arial'>From Windows Explorer, double-click on the file: <b>NCA_Toolbox_for_Windows.exe</b>
     in the &quot;<span class=SpellE>nca</span>&quot; subdirectory. <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l0 level1 lfo2;tab-stops:list .5in'><span style='font-family:
     Arial'>This file loads MATLAB, sets the NCA Toolbox path <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l0 level1 lfo2;tab-stops:list .5in'><span style='font-family:
     Arial'>and automatically executes the MATLAB script:&nbsp; <span
     class=SpellE><b>RunNCAToolbox</b></span><o:p></o:p></span></li>
</ul>

<pre><span style='font-size:12.0pt;font-family:Arial'>If for some reason, this does not work then follow the Unix/Linux instructions.<u1:p></u1:p></span><span
style='font-family:Arial'><o:p></o:p></span></pre>

<p style='tab-stops:45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt'><b><span
style='font-family:Arial'>Instructions for UNIX Systems including Linux and Mac
OS X:<u1:p></u1:p></span></b><span style='font-size:7.0pt;font-family:Arial'><span
style='font-size-adjust: none;font-stretch: normal'><br style='mso-special-character:
line-break'>
<![if !supportLineBreakNewLine]><br style='mso-special-character:line-break'>
<![endif]><o:p></o:p></span></p>

<ul type=disc>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l3 level1 lfo3;tab-stops:list .5in'></span>Unzip the NCA Toolbox
     Release ZIP File and extract all files to a directory.<u1:p></u1:p> </li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l3 level1 lfo3;tab-stops:list .5in'><span style='font-family:
     Arial'>Load the MATLAB program.<u1:p></u1:p> <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l3 level1 lfo3;tab-stops:list .5in'><span style='font-family:
     Arial'>From the command prompt within the MATLAB interpreter change to the
     '<span class=SpellE>nca</span>' subdirectory within the installed NCA
     toolbox directory.&nbsp; <o:p></o:p></span></li>
</ul>

<p style='tab-stops:45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt'><span
style='font-family:Arial'><br>
For example:<u1:p></u1:p> <o:p></o:p></span></p>

<p class=MsoNormal style='tab-stops:45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt'><span
style='font-family:Arial'>&gt;&gt; <span class=SpellE><span class=GramE><b>cd</b></span></span><b>
nca_release_1.0/nca</b><u1:p></u1:p><span style='font-size-adjust: none;
font-stretch: normal'><br>
</span>Execute the NCA Toolbox GUI with the command (ignore the &gt;&gt;) <o:p></o:p></span></p>

<p class=MsoNormal style='tab-stops:45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt'><b><span
style='font-family:Arial'>&gt;&gt; <span class=SpellE>RunNCAToolbox</span></span><u1:p></u1:p></b><span
style='font-family:Arial'> <o:p></o:p></span></p>

<div style='border:none;border-bottom:solid windowtext 1.0pt;mso-border-bottom-alt:
solid windowtext .75pt;padding:0in 0in 1.0pt 0in'>

<p style='tab-stops:45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt'><span
style='font-family:Arial'>Note that the NCA Toolbox will automatically modify
your MATLAB path to ensure the NCA program files load correctly.&nbsp; <br>
To reset your path load <span class=SpellE>pathtool</span> from the command
prompt and click &quot;default&quot; and then &quot;save path&quot;.&nbsp;<o:p></o:p></span></p>

<p style='margin-bottom:12.0pt;tab-stops:45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt'><span
style='font-family:Arial'><br style='mso-special-character:line-break'>
<![if !supportLineBreakNewLine]><br style='mso-special-character:line-break'>
<![endif]><o:p></o:p></span></p>

</div>

<p style='tab-stops:45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt'><b><span
style='font-size:18.0pt;font-family:Arial'>Demo NCA Session<u1:p></u1:p></span></b><span
style='font-family:Arial'><o:p></o:p></span></p>

<p style='tab-stops:45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt'><span
style='font-family:Arial'>To get started:<o:p></o:p></span></p>

<ul type=disc>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l4 level1 lfo4;tab-stops:list .5in'><b><span style='font-family:
     Arial'>Step 1:</span></b><span style='font-family:Arial'>&nbsp; Select <b><i>E.
     coli</i></b> as the organism to analyze in the pop-up box.<u1:p></u1:p> <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l4 level1 lfo4;tab-stops:list .5in'><b><span style='font-family:
     Arial'>Step 2:</span></b><span style='font-family:Arial'>&nbsp; Next click
     &quot;Load Connectivity Matrix&quot; <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l4 level1 lfo4;tab-stops:list .5in'><span style='font-family:
     Arial'>Select the following file by clicking on it: <o:p></o:p></span></li>
</ul>

<ul type=disc>
 <ul type=circle>
  <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
      auto;mso-list:l4 level2 lfo4;tab-stops:list 1.0in'><span class=SpellE><b><span
      style='font-family:Arial'>E_coli_connectivity_Kao_et_al_PNAS.mat</span></b></span><span
      style='font-family:Arial'><o:p></o:p></span></li>
 </ul>
</ul>

<ul type=disc>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l4 level1 lfo4;tab-stops:list .5in'><span style='font-family:
     Arial'>Now click &quot;Load from Selected
     File&quot;.&nbsp;&nbsp;&nbsp;&nbsp; The file should load successfully and
     now close the load screen and you will be returned to the main screen.<u1:p></u1:p>
     <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l4 level1 lfo4;tab-stops:list .5in'><b><span style='font-family:
     Arial'>Step 4:</span></b><span style='font-family:Arial'>&nbsp; Next click
     &quot;Load Microarray Data&quot;.&nbsp;&nbsp; <o:p></o:p></span></li>
</ul>

<ul type=disc>
 <ul type=circle>
  <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
      auto;mso-list:l4 level2 lfo4;tab-stops:list 1.0in'><span
      style='font-family:Arial'>Select the following file by clicking on
      it:&nbsp;&nbsp; <span class=SpellE><b>E_coli_Glucose_to_Acetate_Shift_Kao_et_al_PNAS.mat<u1:p></u1:p></b></span><o:p></o:p></span></li>
 </ul>
</ul>

<ul type=disc>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l4 level1 lfo4;tab-stops:list .5in'><span style='font-family:
     Arial'>Now click:&nbsp; &quot;Load from Selected File&quot;<u1:p></u1:p>.&nbsp;
     The file should load successfully and you will be returned to the main
     screen. <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l4 level1 lfo4;tab-stops:list .5in'><b><span style='font-family:
     Arial'>Step 5</span></b><span style='font-family:Arial'>. Now click
     &quot;Match Microarray Data and Connectivity&quot;. <o:p></o:p></span></li>
</ul>

<ul type=disc>
 <ul type=circle>
  <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
      auto;mso-list:l4 level2 lfo4;tab-stops:list 1.0in'><span
      style='font-family:Arial'>This will match the connectivity matrix to the
      microarray data matrix row by row by each identifier.&nbsp; A common
      error is that the selected organism (in Step 1) is wrong, so the matching
      fails.&nbsp; Make sure to double check that you selected the
      correct.&nbsp;<o:p></o:p></span></li>
 </ul>
</ul>

<ul type=disc>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l4 level1 lfo4;tab-stops:list .5in'><b><span style='font-family:
     Arial'>Step 5:</span></b><span style='font-family:Arial'>&nbsp; Now click
     &quot;Select Regulatory Network&quot;<u1:p></u1:p>.&nbsp; <o:p></o:p></span></li>
</ul>

<ul type=disc>
 <ul type=circle>
  <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
      auto;mso-list:l4 level2 lfo4;tab-stops:list 1.0in'><span
      style='font-family:Arial'>To choose a regulatory network:&nbsp; Select
      &quot;16 <span class=SpellE>TFs</span>&quot; and click auto-fill.<u1:p></u1:p>
      Then Click &quot;Find NCA Network&quot;<u1:p></u1:p>&nbsp; <o:p></o:p></span></li>
  <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:
      auto;mso-list:l4 level2 lfo4;tab-stops:list 1.0in'><span
      style='font-family:Arial'>Now Click &quot;Save Selected <span
      class=SpellE>Subnetwork</span> &quot;.&nbsp; This will save the selected <span
      class=SpellE>subnetwork</span> to the global workspace.<o:p></o:p></span></li>
 </ul>
</ul>

<u1:p></u1:p>

<ul type=disc>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l4 level1 lfo4;tab-stops:list .5in'><b><span style='font-family:
     Arial'>Final Step:</span></b><span style='font-family:Arial'>&nbsp; Now
     Click the Button &quot;Run NCA&quot;.<u1:p></u1:p> <o:p></o:p></span></li>
</ul>

<p class=MsoNormal style='tab-stops:45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt'><span
style='font-family:Arial'>Now you will see a progress bar indicating the total
number of initial guess iterations being run.&nbsp; After NCA completes running
you can explore the TFA profiles by graphical <br>
analysis via the plot menu.&nbsp; You may also cluster the TFAs using the
Analyze TFA menu.&nbsp; At any time you can save your workspace or load a
previous workspace from the file menu.<u1:p></u1:p> <o:p></o:p></span></p>

<pre><b><span style='font-size:18.0pt;font-family:Arial'><o:p>&nbsp;</o:p></span></b></pre><pre><b><span
style='font-size:18.0pt;font-family:Arial'>Spectra Analysis</span></b><b><span
style='font-size:12.0pt;font-family:Arial'><br style='mso-special-character:
line-break'>
<![if !supportLineBreakNewLine]><br style='mso-special-character:line-break'>
<![endif]><o:p></o:p></span></b></pre><pre><b><span style='font-size:12.0pt;
font-family:Arial'><o:p>&nbsp;</o:p></span></b></pre><pre><span
style='font-family:Arial'>First we assume that you have successfully run the demo session of NCA.<span style='mso-spacerun:yes'>  </span>If you have not, please go to the demo tutorial of this document.<span style='mso-spacerun:yes'>  </span>Spectra analysis assumes that you have provided pre-matched experimental and connectivity data.<span style='mso-spacerun:yes'>  </span>This means that the matrices are lined up row-by-row and there is exactly the same number of rows in both the connectivity and expression matrices.<br
style='mso-special-character:line-break'>
<![if !supportLineBreakNewLine]><br style='mso-special-character:line-break'>
<![endif]><o:p></o:p></span></pre><pre><span style='font-family:Arial'><o:p>&nbsp;</o:p></span></pre><pre><span
style='font-family:Arial'>To get started, select as your organism type:&nbsp; &quot;Spectra&quot;.&nbsp; Then click &quot;Match Spectra Data to Connectivity Data&quot;, followed by &quot;Select Regulatory Network&quot;.<span style='mso-spacerun:yes'>    </span>Here you can choose the (regulatory) factors you want to analyze.<span style='mso-spacerun:yes'>  </span>Save your selection to the workspace and then run NCA.<span style='mso-spacerun:yes'>   </span>Now, the program will ask you for your choice of normalization.<span style='mso-spacerun:yes'>  </span>Here you should enter the absorption vector (space delimited) one value for each factor, and the index of the wavelength (P) vector to normalize by.<span style='mso-spacerun:yes'>  </span>This should be an integer greater than 0 and less than the total number of experiments.<br
style='mso-special-character:line-break'>
<![if !supportLineBreakNewLine]><br style='mso-special-character:line-break'>
<![endif]><o:p></o:p></span></pre><pre><span style='font-family:Arial'><o:p>&nbsp;</o:p></span></pre><pre><span
style='font-family:Arial'>For example, the index in Figure 1 is set to 1 <span
class=SpellE>1</span> <span class=SpellE>1</span> which is the first indexed value of the decomposed spectrum.<span style='mso-spacerun:yes'>  </span>The matching absorptions for this index are 0.3968 0.2736 0.3365. <o:p></o:p></span></pre><pre><span
style='font-family:Arial'><o:p>&nbsp;</o:p></span></pre><pre><span
style='font-family:Arial'>Now click OK to use these normalization settings, or click CANCEL to use the <span
class=GramE>default<span style='mso-spacerun:yes'>  </span>microarray</span> normalization <span
class=SpellE>proceedure</span>.<span style='mso-spacerun:yes'>  </span>Generally you want to click OK if you are analyzing the hemoglobin spectra from the PNAS paper.<o:p></o:p></span></pre><pre><span
style='font-family:Arial'><span style='mso-spacerun:yes'>      </span><br
style='mso-special-character:line-break'>
<![if !supportLineBreakNewLine]><br style='mso-special-character:line-break'>
<![endif]><o:p></o:p></span></pre><pre><span style='font-family:Arial'><o:p>&nbsp;</o:p></span></pre><pre><span
style='font-family:Arial'><o:p>&nbsp;</o:p></span></pre><pre><span
style='font-family:Arial'><o:p>&nbsp;</o:p></span></pre><pre><span
style='font-family:Arial'><o:p>&nbsp;</o:p></span></pre><pre><b><span
style='font-size:12.0pt;font-family:Arial'>Figure 1:<span style='mso-spacerun:yes'>  </span>Spectrum Normalization</span></b><span
style='font-size:12.0pt;font-family:Arial'><br style='mso-special-character:
line-break'>
<![if !supportLineBreakNewLine]><br style='mso-special-character:line-break'>
<![endif]><o:p></o:p></span></pre><pre><span style='font-size:12.0pt;
font-family:Arial'><o:p>&nbsp;</o:p></span></pre><pre><span style='font-size:
12.0pt;font-family:Arial'><img border=0 width=1255 height=483 id="_x0000_i1026"
src="tutorial_figure1.PNG" alt="Picture of Spectrum Normalization"><br
style='mso-special-character:line-break'>
<![if !supportLineBreakNewLine]><br style='mso-special-character:line-break'>
<![endif]><o:p></o:p></span></pre><pre><span style='font-size:12.0pt;
font-family:Arial'><o:p>&nbsp;</o:p></span></pre><pre><span style='font-size:
12.0pt;font-family:Arial'>After running NCA you can plot the spectra from the Plot menu. <o:p></o:p></span></pre>

<p class=MsoNormal style='tab-stops:45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt'><b><span
style='font-size:13.5pt;font-family:Arial'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNormal style='tab-stops:45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt'><b><span
style='font-size:13.5pt;font-family:Arial'>Connectivity and Expression Data
Examples:<u1:p></u1:p></span></b><span style='font-family:Arial'><o:p></o:p></span></p>

<p style='tab-stops:45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt'><span
style='font-family:Arial'><u1:p></u1:p>We have provided pre-initialized data
for the NCA Toolbox.&nbsp; This data consists <br>
of a regulatory network data or expression data (signal data), and is available
for <br>
Spectra,&nbsp;Escherichia<i> coli K12</i>, <span class=SpellE><i>Saccharomyces</i></span><i>
<span class=SpellE>cerevisiae</span></i> and <i>Homo sapiens</i>.&nbsp; <br>
These can be loaded directly from the Toolbox and are visible in the list box
of the <br>
load connectivity or load microarray screens.<u1:p></u1:p><o:p></o:p></span></p>

<p class=MsoNormal style='tab-stops:45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt'><b><span
style='font-family:Arial'>&nbsp;</span></b><b><span style='font-size:13.5pt;
font-family:Arial'><u1:p></u1:p>Input Data Formats:<u1:p></u1:p></span></b><span
style='font-family:Arial'> <o:p></o:p></span></p>

<p style='tab-stops:45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt'><span
style='font-family:Arial'>The NCA Toolbox reads tab-delimited text files that
are organized <br>
in a specific row-column format.&nbsp; That is, you should include a column
header and row header for each file.&nbsp; Example data files for both
connectivity and expression <br>
are provided in the <span class=SpellE><b>ExampleData</b></span> directory.<br>
&nbsp; <br>
You <b>cannot have</b> <b>missing values </b>in either the microarray or
connectivity file and the Toolbox should warn you if you do.&nbsp; To fix this,
you should remove any genes (rows) that have missing data before running NCA.&nbsp;
Alternatively, you can impute the missing values using standard preprocessing
tools such as GEPAS: <a href="http://gepas.bioinfo.cnio.es/tools.html">http://gepas.bioinfo.cnio.es/tools.html</a>
, <br>
save this data to a new file.&nbsp; After running the GEAPS tools you first,
have to load the file <span class=GramE>into<span style='mso-spacerun:yes'> 
</span>MS</span> Excel and remove any comment lines beginning with a hash/pound
sign ( symbol # ).<span style='mso-spacerun:yes'>  </span>Save this new file in
tab delimited format, and then load it into the NCA Toolbox.&nbsp; Again,
example files are provided in the <span class=SpellE><b>ExampleData</b></span><b>.</b><u1:p></u1:p><o:p></o:p></span></p>

<p class=MsoNormal style='tab-stops:45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt'><b><u1:p></u1:p><span
style='font-family:Arial'>Note<span class=GramE>:</span></span></b><span
style='font-family:Arial'><br>
GEPAS is a web-based program to filter or impute missing values.&nbsp; <br>
We recommend k nearest neighbor (<span class=SpellE>kNN</span>) imputation and
to set k=15. <br>
<br>
<b><u1:p></u1:p>Setting TF Constraints:</b> <br>
<br>
NCA allows you to specify zero constraints on the TFA profiles, which <br>
is necessary, for example, when analyzing expression profiling of a regulator
knockout. <br>
<br>
To select TF constraints, after loading in the data and connectivity matrices,
matching them, and selecting the regulatory network you want to analyze, then <span
class=GramE>click</span> Set TF constraints. <br>
<br>
<b style='mso-bidi-font-weight:normal'>Step-by-step: <o:p></o:p></b></span></p>

<p class=MsoNormal style='tab-stops:45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt'><span
style='font-family:Arial'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='tab-stops:45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt'><span
style='font-family:Arial'>Click on Set TF constraints. <o:p></o:p></span></p>

<ul type=disc>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l1 level1 lfo5;tab-stops:list .5in'><span style='font-family:
     Arial'>Add each TF and experiment pair to the list of available
     constraints. <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l1 level1 lfo5;tab-stops:list .5in'><span style='font-family:
     Arial'>Click Save Constraints to save the regulator knockout constraints
     to the global workspace. <o:p></o:p></span></li>
 <li class=MsoNormal style='mso-margin-top-alt:auto;mso-margin-bottom-alt:auto;
     mso-list:l1 level1 lfo5;tab-stops:list .5in'><span style='font-family:
     Arial'>Now run NCA. <o:p></o:p></span></li>
</ul>

<p class=MsoNormal style='tab-stops:45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt'><b><span
style='font-family:Arial'>How to Analyze an Organism not in the list<span
class=GramE>:</span></span></b><span style='font-family:Arial'><br>
<br>
With NCA you can analyze other organisms by providing <span class=SpellE>prematched</span><br>
connectivity and expression data. Select your organism of interest<span
class=GramE>,</span><br>
then click the check box &#8220;Check if Connectivity and Microarray data are <span
class=SpellE>prematched</span> by Row&#8221;. Then click Match Data Matrices
which will copy over the <span class=SpellE>prematched</span> data for NCA
analysis. <span style='mso-spacerun:yes'> </span>You can select and analyze
different regulatory networks using the &quot;Select Regulators&quot; option. <o:p></o:p></span></p>

<p class=MsoNormal style='tab-stops:45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt'><span
style='font-family:Arial'><br>
<b>Inverting TFA Profiles to Match Known Regulations<o:p></o:p></b></span></p>

<p class=MsoNormal style='tab-stops:45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt'><span
style='font-family:Arial'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='tab-stops:45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt'><span
style='font-family:Arial'>As a scientist you may have prior knowledge of the sign
of a TFA.<span style='mso-spacerun:yes'>  </span>For instance you may expect
the regulatory activity to increase positively and then decrease eventually
hitting a steady state of zero activity.<span style='mso-spacerun:yes'> 
</span>However, the NCA solution may in fact be the exact opposite.<span
style='mso-spacerun:yes'>  </span>This is not cause for concern.<span
style='mso-spacerun:yes'>  </span>These solutions are in fact identical, except
that the negative has been placed on the TFA profile rather then the CS value
during optimization.<span style='mso-spacerun:yes'>  </span>This happens
mathematically, because the gene expression profile is fit as the product of CS
and TFA profile.<span style='mso-spacerun:yes'>  </span>To fix this we provide
a menu screen to change each individual TFA profile.<span
style='mso-spacerun:yes'>  </span>This allows you to inspect each TFA profile
and modify them accordingly.<span style='mso-spacerun:yes'>  </span>Remember
that inverting a TFA profile (or flipping) does not change the solution.<o:p></o:p></span></p>

<p class=MsoNormal style='tab-stops:45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt'><span
style='font-family:Arial'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='tab-stops:45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt'><b><span
style='font-family:Arial'>A Note on Microarray and Connectivity Data Format</span></b><span
style='font-family:Arial'><br>
<br>
NCA uses ORF label identifiers for <i>E. coli</i> and <i>S. <span class=SpellE>cerevisiae</span></i>
to match expression data with connectivity data. You can sometimes use the
common gene name to match identifiers. For <i>H. sapiens</i> we use the <span
class=SpellE>Ensembl</span> identifier, the NCBI <span class=SpellE>Genbank</span>
identifier, or the common gene name. Microarray and expression data are matched
together (that is lined up row-by-row) using these labels to find the common
index. To explore the labels we have in our <span class=SpellE>Genebank</span>
database, each <span class=SpellE>genebank</span> can be explored by loading it
into&nbsp;the MATLAB spreadsheet from the subdirectory: <span class=SpellE><b>nca/src/genebanks</b></span>.
However, if you are going to do this, please make a backup of the software.<b><br
style='mso-special-character:line-break'>
<![if !supportLineBreakNewLine]><br style='mso-special-character:line-break'>
<![endif]><o:p></o:p></b></span></p>

<p class=MsoNormal style='tab-stops:45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt'><b><span
style='font-family:Arial'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNormal style='tab-stops:45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt'><b><span
style='font-family:Arial'>Fast NCA<o:p></o:p></span></b></p>

<p class=MsoNormal style='tab-stops:45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt'><b><span
style='font-family:Arial'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNormal style='tab-stops:45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt'><span
style='font-family:Arial;mso-bidi-font-weight:bold'>The Fast NCA option allows
the solution to converge to a more relaxed solution by increasing the condition
number approximation of the second NCA criteria as determined during
optimization.<span style='mso-spacerun:yes'>  </span>Fast NCA can be used to
quickly assess the convergence properties of an analysis, but for final use,
regularized NCA or normal NCA should be used.<o:p></o:p></span></p>

<p class=MsoNormal style='tab-stops:45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt'><span
style='font-family:Arial'><br>
<b>How to Get Help</b><br>
<br>
Software updates and new connectivity information will be present at
http://www.seas.ucla.edu/~liaoj.<br>
For software help please email Simon Galbraith sgalbrai@cs.ucla.edu. We are
also working on a version <br>
of this toolbox for non-MATLAB users.<br>
<br>
<b>Regularization Toolkit</b><br>
<br>
The regularization toolkit is authored by Prof. <span class=GramE>Per</span>
Christian Hansen, and is available for download at <a
href="http://www2.imm.dtu.dk/~pch/Regutools/">http://www2.imm.dtu.dk/~pch/Regutools/</a>
<br style='mso-special-character:line-break'>
<![if !supportLineBreakNewLine]><br style='mso-special-character:line-break'>
<![endif]><o:p></o:p></span></p>

<p class=MsoNormal style='tab-stops:45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt'><span
style='font-family:Arial'>P. C. Hansen, <i>Regularization Tools: A MATLAB package
for analysis and <br>
solution of discrete ill-posed problems</i>, Numerical Algorithms <b>6</b> <br>
(1994), pp. 1-35. <br>
<br>
<span class=GramE>and</span><o:p></o:p></span></p>

<p style='tab-stops:45.8pt 91.6pt 137.4pt 183.2pt 229.0pt 274.8pt 320.6pt 366.4pt 412.2pt 458.0pt 503.8pt 549.6pt 595.4pt 641.2pt 687.0pt 732.8pt'><span
style='font-family:Arial'>P. C. Hansen, <i>Rank-Deficient and Discrete
Ill-Posed Problems: Numerical Aspects of Linear Inversion</i>, <st1:country-region
w:st="on">SIAM</st1:country-region>, <st1:City w:st="on"><st1:place w:st="on">Philadelphia</st1:place></st1:City>,
1998. <o:p></o:p></span></p>

<pre style='margin-bottom:12.0pt'><span style='font-family:Arial'><br>
</span><b><span style='font-size:12.0pt;font-family:Arial'>Description of MATLAB Variables in the Global Workspace</span></b><span
style='font-family:Arial'><o:p></o:p></span></pre>

<table class=MsoNormalTable border=1 cellpadding=0 width="50%"
 style='width:50.0%;mso-cellspacing:1.5pt;mso-padding-alt:1.5pt 1.5pt 1.5pt 1.5pt'>
 <tr style='mso-yfti-irow:0;mso-yfti-firstrow:yes'>
  <td style='padding:1.5pt 1.5pt 1.5pt 1.5pt'>
  <p class=MsoNormal><span style='font-family:Arial'>Variable Name<o:p></o:p></span></p>
  </td>
  <td style='padding:1.5pt 1.5pt 1.5pt 1.5pt'>
  <p class=MsoNormal><b><span style='font-family:Arial'>Description of Variable</span></b><span
  style='font-family:Arial'><o:p></o:p></span></p>
  </td>
 </tr>
 <tr style='mso-yfti-irow:1'>
  <td style='padding:1.5pt 1.5pt 1.5pt 1.5pt'>
  <p class=MsoNormal><span class=SpellE><span style='font-family:Arial'>Ap</span></span><span
  style='font-family:Arial'><o:p></o:p></span></p>
  </td>
  <td style='padding:1.5pt 1.5pt 1.5pt 1.5pt'>
  <p class=MsoNormal><span style='font-family:Arial'>Final NCA computer values
  of CS over the selected regulators<o:p></o:p></span></p>
  </td>
 </tr>
 <tr style='mso-yfti-irow:2'>
  <td style='padding:1.5pt 1.5pt 1.5pt 1.5pt'>
  <p class=MsoNormal><span style='font-family:Arial'>P<o:p></o:p></span></p>
  </td>
  <td style='padding:1.5pt 1.5pt 1.5pt 1.5pt'>
  <p class=MsoNormal><span style='font-family:Arial'>Final NCA computed TFA
  profiles<o:p></o:p></span></p>
  </td>
 </tr>
 <tr style='mso-yfti-irow:3'>
  <td style='padding:1.5pt 1.5pt 1.5pt 1.5pt'>
  <p class=MsoNormal><span class=SpellE><span style='font-family:Arial'>A_rank_final</span></span><span
  style='font-family:Arial'><o:p></o:p></span></p>
  </td>
  <td style='padding:1.5pt 1.5pt 1.5pt 1.5pt'>
  <p class=MsoNormal><span style='font-family:Arial'>The structured matrix that
  satisfies NCA criteria and can be used to run NCA<o:p></o:p></span></p>
  </td>
 </tr>
 <tr style='mso-yfti-irow:4'>
  <td style='padding:1.5pt 1.5pt 1.5pt 1.5pt'>
  <p class=MsoNormal><span style='font-family:Arial'>E<o:p></o:p></span></p>
  </td>
  <td style='padding:1.5pt 1.5pt 1.5pt 1.5pt'>
  <p class=MsoNormal><span style='font-family:Arial'>Original Expression Data<o:p></o:p></span></p>
  </td>
 </tr>
 <tr style='mso-yfti-irow:5'>
  <td style='padding:1.5pt 1.5pt 1.5pt 1.5pt'>
  <p class=MsoNormal><span class=SpellE><span style='font-family:Arial'>Elabels</span></span><span
  style='font-family:Arial'><o:p></o:p></span></p>
  </td>
  <td style='padding:1.5pt 1.5pt 1.5pt 1.5pt'>
  <p class=MsoNormal><span style='font-family:Arial'>The row labels for matrix
  E<o:p></o:p></span></p>
  </td>
 </tr>
 <tr style='mso-yfti-irow:6'>
  <td style='padding:1.5pt 1.5pt 1.5pt 1.5pt'>
  <p class=MsoNormal><span style='font-family:Arial'>A<o:p></o:p></span></p>
  </td>
  <td style='padding:1.5pt 1.5pt 1.5pt 1.5pt'>
  <p class=MsoNormal><span style='font-family:Arial'>The original connectivity
  data<o:p></o:p></span></p>
  </td>
 </tr>
 <tr style='mso-yfti-irow:7'>
  <td style='padding:1.5pt 1.5pt 1.5pt 1.5pt'>
  <p class=MsoNormal><span class=SpellE><span style='font-family:Arial'>Alabels</span></span><span
  style='font-family:Arial'><o:p></o:p></span></p>
  </td>
  <td style='padding:1.5pt 1.5pt 1.5pt 1.5pt'>
  <p class=MsoNormal><span style='font-family:Arial'>The row labels for matrix
  A<o:p></o:p></span></p>
  </td>
 </tr>
 <tr style='mso-yfti-irow:8'>
  <td style='padding:1.5pt 1.5pt 1.5pt 1.5pt'>
  <p class=MsoNormal><span class=SpellE><span style='font-family:Arial'>Plabels</span></span><span
  style='font-family:Arial'><o:p></o:p></span></p>
  </td>
  <td style='padding:1.5pt 1.5pt 1.5pt 1.5pt'>
  <p class=MsoNormal><span style='font-family:Arial'>The column labels for
  matrix A<o:p></o:p></span></p>
  </td>
 </tr>
 <tr style='mso-yfti-irow:9'>
  <td style='padding:1.5pt 1.5pt 1.5pt 1.5pt'>
  <p class=MsoNormal><span class=SpellE><span style='font-family:Arial'>Experiment_labels</span></span><span
  style='font-family:Arial'><o:p></o:p></span></p>
  </td>
  <td style='padding:1.5pt 1.5pt 1.5pt 1.5pt'>
  <p class=MsoNormal><span style='font-family:Arial'>The column labels for
  matrix E<o:p></o:p></span></p>
  </td>
 </tr>
 <tr style='mso-yfti-irow:10'>
  <td style='padding:1.5pt 1.5pt 1.5pt 1.5pt'>
  <p class=MsoNormal><span class=SpellE><span style='font-family:Arial'>tfa_ids</span></span><span
  style='font-family:Arial'><o:p></o:p></span></p>
  </td>
  <td style='padding:1.5pt 1.5pt 1.5pt 1.5pt'>
  <p class=MsoNormal><span style='font-family:Arial'>The subset of <span
  class=SpellE>Plabels</span> that has connectivity in A for the genes found in
  <span class=SpellE>Elabels</span><o:p></o:p></span></p>
  </td>
 </tr>
 <tr style='mso-yfti-irow:11'>
  <td style='padding:1.5pt 1.5pt 1.5pt 1.5pt'>
  <p class=MsoNormal><span class=SpellE><span style='font-family:Arial'>gene_ids</span></span><span
  style='font-family:Arial'><o:p></o:p></span></p>
  </td>
  <td style='padding:1.5pt 1.5pt 1.5pt 1.5pt'>
  <p class=MsoNormal><span style='font-family:Arial'>The subset of <span
  class=SpellE>Elabels</span> that matches the connectivity labels in matrix A<o:p></o:p></span></p>
  </td>
 </tr>
 <tr style='mso-yfti-irow:12'>
  <td style='padding:1.5pt 1.5pt 1.5pt 1.5pt'>
  <p class=MsoNormal><span style='font-family:Arial'>method<o:p></o:p></span></p>
  </td>
  <td style='padding:1.5pt 1.5pt 1.5pt 1.5pt'>
  <p class=MsoNormal><span style='font-family:Arial'>The selected NCA
  method:&nbsp; <span class=SpellE>gNCA</span> or NCA<o:p></o:p></span></p>
  </td>
 </tr>
 <tr style='mso-yfti-irow:13'>
  <td style='padding:1.5pt 1.5pt 1.5pt 1.5pt'>
  <p class=MsoNormal><span style='font-family:Arial'>organism<o:p></o:p></span></p>
  </td>
  <td style='padding:1.5pt 1.5pt 1.5pt 1.5pt'>
  <p class=MsoNormal><span style='font-family:Arial'>The selected organism to
  analyze<o:p></o:p></span></p>
  </td>
 </tr>
 <tr style='mso-yfti-irow:14'>
  <td style='padding:1.5pt 1.5pt 1.5pt 1.5pt'>
  <p class=MsoNormal><span style='font-family:Arial'>iterations<o:p></o:p></span></p>
  </td>
  <td style='padding:1.5pt 1.5pt 1.5pt 1.5pt'>
  <p class=MsoNormal><span style='font-family:Arial'>The default number of
  iterations to run NCA before averaging and return a solution.<o:p></o:p></span></p>
  </td>
 </tr>
 <tr style='mso-yfti-irow:15'>
  <td style='padding:1.5pt 1.5pt 1.5pt 1.5pt'>
  <p class=MsoNormal><span style='font-family:Arial'>E0<o:p></o:p></span></p>
  </td>
  <td style='padding:1.5pt 1.5pt 1.5pt 1.5pt'>
  <p class=MsoNormal><span style='font-family:Arial'>The subset of E that
  matches the connectivity data<o:p></o:p></span></p>
  </td>
 </tr>
 <tr style='mso-yfti-irow:16'>
  <td style='padding:1.5pt 1.5pt 1.5pt 1.5pt'>
  <p class=MsoNormal><span style='font-family:Arial'>A0<o:p></o:p></span></p>
  </td>
  <td style='padding:1.5pt 1.5pt 1.5pt 1.5pt'>
  <p class=MsoNormal><span style='font-family:Arial'>The subset of A that
  matches the expression (or signal) data<o:p></o:p></span></p>
  </td>
 </tr>
 <tr style='mso-yfti-irow:17'>
  <td style='padding:1.5pt 1.5pt 1.5pt 1.5pt'>
  <p class=MsoNormal><span class=SpellE><span style='font-family:Arial'>gene_ids_final</span></span><span
  style='font-family:Arial'><o:p></o:p></span></p>
  </td>
  <td style='padding:1.5pt 1.5pt 1.5pt 1.5pt'>
  <p class=MsoNormal><span style='font-family:Arial'>The set of gene labels
  that correspond to the rows in <span class=SpellE>A_rank_final</span> <o:p></o:p></span></p>
  </td>
 </tr>
 <tr style='mso-yfti-irow:18'>
  <td style='padding:1.5pt 1.5pt 1.5pt 1.5pt'>
  <p class=MsoNormal><span class=SpellE><span style='font-family:Arial'>tfa_ids_final</span></span><span
  style='font-family:Arial'><o:p></o:p></span></p>
  </td>
  <td style='padding:1.5pt 1.5pt 1.5pt 1.5pt'>
  <p class=MsoNormal><span style='font-family:Arial'>The index vector of <span
  class=SpellE>tfa_ids</span> that corresponds to the columns of <span
  class=SpellE>A_rank_final</span><o:p></o:p></span></p>
  </td>
 </tr>
 <tr style='mso-yfti-irow:19'>
  <td style='padding:1.5pt 1.5pt 1.5pt 1.5pt'>
  <p class=MsoNormal><span class=SpellE><span style='font-family:Arial'>Ep</span></span><span
  style='font-family:Arial'><o:p></o:p></span></p>
  </td>
  <td style='padding:1.5pt 1.5pt 1.5pt 1.5pt'>
  <p class=MsoNormal><span style='font-family:Arial'>The subset of expression
  data (E0) used to compute the final matrices <span class=SpellE>Ap</span> and
  P. <o:p></o:p></span></p>
  </td>
 </tr>
 <tr style='mso-yfti-irow:20;mso-yfti-lastrow:yes'>
  <td style='padding:1.5pt 1.5pt 1.5pt 1.5pt'>
  <p class=MsoNormal><span class=SpellE><span style='font-family:Arial'>P_dev</span></span><span
  style='font-family:Arial'><o:p></o:p></span></p>
  </td>
  <td style='padding:1.5pt 1.5pt 1.5pt 1.5pt'>
  <p class=MsoNormal><span style='font-family:Arial'>Matrix of TFA deviations<o:p></o:p></span></p>
  </td>
 </tr>
</table>

<pre><span style='font-family:Arial'><o:p>&nbsp;</o:p></span></pre></div>

</div>

</body>

</html>
